css/index.css

```diff
@@ -111,6 +146,37 @@ p {
     align-items: center;
     justify-content: center;
 }
```

Comment:
Since you created a class for each one box1, box2, box3, etc. you can target the class directly without having to use .middle-content .boxes.